data/json/items/gunmod/underbarrel.json

```diff
@@ -610,5 +610,26 @@
       "moves": 150
     },
     "flags": [ "BIPOD", "SLOW_WIELD", "ZERO_WEIGHT" ]
+  },
+  {
+    "id": "muzzle_weight",
+    "type": "GUNMOD",
+    "name": { "str": "muzzle weight" },
+    "description": "A small weight attached below the barrel of a firearm to reduce muzzle climb, increasing bulk but reducing shot spread.  Mainly used on competition handuns, but it can be added to combat pistols, too.",
```

Comment:
```suggestion
    "description": "A small weight attached below the barrel of a firearm to reduce muzzle climb, increasing bulk but reducing shot spread.  Mainly used on competition handguns, but it can be added to combat pistols, too.",
```